lib/src/models/result_movie.dart

```diff
@@ -54,20 +54,32 @@ class ResultMovie {
     return ResultMovie(
       voteCount: parsedJson['vote_count'],
       id: parsedJson['id'],
-      video: parsedJson['video'],
-      voteAverage: parsedJson['vote_average'],
-      title: parsedJson['title'],
+      video: parsedJson['video'] != null
```

Comment:
Take a look at this null operator: https://news.dartlang.org/2015/08/dart-112-released-with-null-aware.html, this can make you save coding time a it will make your code easy to read

Reply:
Done, thank you very much for the tip. Also about the comment above I have to say that I've tried to implement the json_serializable package but I couldn't make it work, but I'll take that advise for the future for sure, thank you for that too.